app/views/users/piv_cac_setup_from_sign_in/prompt.html.erb

```diff
@@ -10,23 +10,19 @@
   <%= t('instructions.mfa.piv_cac.add_from_sign_in_html') %>
 </p>
 
-<div class="margin-top-4">
-  <div class="inline-block">
-    <%= form_tag(submit_new_piv_cac_url) do %>
-      <%= label_tag 'code', t('forms.piv_cac_setup.nickname'), class: 'block bold' %>
-      <%= text_field_tag :name, '', required: true, aria: { invalid: false }, id: 'nickname',
-                                    class: 'block col-12 field font-family-mono', size: 16, maxlength: 20,
-                                    'aria-labelledby': 'totp-label' %>
-       <div class="margin-top-2">
-        <%= submit_tag t('forms.piv_cac_setup.submit'), class: 'usa-button usa-button--wide usa-button--big' %>
-      </div>
-      <div class="margin-top-2">
-        <%= link_to t('forms.piv_cac_setup.no_thanks'),
-                    new_user_session_url,
-                    class: 'usa-button usa-button--wide usa-button--big usa-button--outline' %>
-      </div>
-    <% end %>
+<%= form_tag(submit_new_piv_cac_url, class: 'margin-top-4') do %>
+  <%= label_tag 'code', t('forms.piv_cac_setup.nickname'), class: 'block bold' %>
+  <%= text_field_tag :name, '', required: true, aria: { invalid: false }, id: 'nickname',
+                                class: 'block col-12 field font-family-mono', size: 16, maxlength: 20,
+                                'aria-labelledby': 'totp-label' %>
```

Comment:
looks like this was existing, but is there a reason `aria-labelledby` isn't inside the other aria block?
```suggestion
  <%= text_field_tag :name, '', required: true, aria: { invalid: false, labelledby: 'totp-label' }, id: 'nickname',
                                class: 'block col-12 field font-family-mono', size: 16, maxlength: 20 %>
```

Reply:
> but is there a reason `aria-labelledby` isn't inside the other aria block?

No, it could have been in the `aria` block. But on closer inspection, there's quite a few other issues related to this. Specifically, the `aria-labelledby` is referencing a label which doesn't exist. The input gives itself an ID `nickname`, but the preceding label references an ID `code`. So all said and done, the input isn't labelled ([bad](https://www.w3.org/WAI/WCAG21/Techniques/failures/F68.html)).

Cleaned it up in 308ecbc86.